shotgun_api3/lib/httplib2/__init__.py

```diff
@@ -9,24 +7,13 @@
 # current python version.  httplib2 supports python 2/3 by forking the code rather
 # than with a single cross-compatible module. Rather than modify third party code,
 # we'll just import the appropriate branch here.
-if six.PY3:
-    # Generate ssl_error_classes
-    import ssl as __ssl
-    ssl_error_classes = (__ssl.SSLError, __ssl.CertificateError)
-    del __ssl
-
-    # get the python3 fork of httplib2
-    from . import python3 as __httplib2_compat
-
-
-else:
-    # Generate ssl_error_classes
-    from .python2 import SSLHandshakeError as __SSLHandshakeError  # TODO: shouldn't rely on this. not public
-    ssl_error_classes = (__SSLHandshakeError,)
-    del __SSLHandshakeError
+# Generate ssl_error_classes
+import ssl as __ssl
+ssl_error_classes = (__ssl.SSLError, __ssl.CertificateError)
```

Comment:
Do we need to keep this variable or can we just replace the code where ssl_error_classes is used by __ssl.SSLError, __ssl.CertificateError?